frontend/src/app/score-board/components/filter-settings/components/score-board-additional-settings-dialog/score-board-additional-settings-dialog.component.scss

```
@@ -41,3 +45,26 @@ h1 {
 button mat-icon {
   margin-right: 8px;
 }
+
+h2 {
+  color: var(--theme-text);
+}
+
+span {
+  color: var(--theme-text);
+  font-size: 14px;
+}
+
+mat-icon {
+  font-size: 24px !important; /* Increases the icon size */
+  height: 24px !important;
+  width: 24px !important;
+}
+
+.tooltip {
+  font-size: 14px !important; /* Increases text size */
```

doesn't need the important or the comment.

```suggestion
  font-size: 14px;
```